packages/commons-ui/src/RichTypography/index.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+import { Typography } from "@material-ui/core";
```

Comment:
Are we using mui4 or mui5 for monorepo?

Reply:
@gertie-sheshe We are using mui5 ....I'll be adding that shortly. The mui4 was only for the setup